src/components/ContactUsForm/ContactUs.jsx

```diff
@@ -33,38 +94,226 @@ const ContactUs = () => {
       been_dismissed: false,
     };
     const result = await backend.post('/notification/', updatedFormData);
+    console.log(result);
+    setShowConfirmation(true);
+  };
+
+  const handleNumInputChange = (e, index) => {
+    checkedItems[index] = Number(e);
+
+    // Update the state with the new array
+    setCheckedItems(checkedItems);
+    isFormFilled();
+    setIsFormValid(checkedItems.some(value => value !== 0) && text.length > 0);
```

Comment:
make this an or so that businesses can request supplies without a message or send a message without requesting supplies